src/components/05_pages/NodeForm/index.js

```diff
@@ -189,6 +189,9 @@ class NodeForm extends React.Component {
                         onChange: (propType === 'attributes'
                           ? this.onAttributeChange
                           : this.onRelationshipChange)(fieldName),
+                        required: this.props.schema.properties.data.properties.attributes.required.includes(
```

Comment:
This doesn't take into account yet required relationship fields. However, it seems like schemata don't expose this information yet so I opened an issue to their queue:  https://www.drupal.org/project/schemata/issues/2991871 . Maybe we should just add a todo comment?